src/client/public-openfin/launcher.json

```diff
@@ -21,7 +21,7 @@
     }
   },
   "runtime": {
-    "version": "17.85.53.10",
+    "version": "22.94.67.8",
```

Comment:
don't check in please .. affects the main app, which is not tested on this version .. will get increased to V28/29 by OF upgrade very soon anyway